### PR TITLE
Tarea 1: Revista empire

### PR DESCRIPTION
**Pull request by tatisgom27**
Hola Adjunto tarea 1

### CODE REVIEW

File: tatiana-gomez-restrepo/tarea-1/CSS/estilos.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+.background {
+    background-color: #dd9d39;
+    margin: 25px;
+    padding: 20px;
+    box-sizing: content-box;
+    width: 100vh;
+  }
+  
+  .titulosuperior {
+    width: 100%;
```
Comment (emolinaglobant) [inline tatiana-gomez-restrepo/tarea-1/CSS/estilos.css:10]: @tatisgom27 , please make sure all CSS properties are alphabetically sorted

File: tatiana-gomez-restrepo/tarea-1/empire.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      <span><strong> FISHER IS NOT BROKEN ///</strong></span>
+      <span class="blanco">THE JUSTICE LEAGUE ENDS HIS SILENCE</span>
+    </header>
+    <div class="contenedor">
+      <img src="img/Logo.png" alt="empire" class="titulo" />
+      <img src="img/imagen.png" alt="loki" class="imgloki" />
+      <time datetime="06/2021" class="fecha"></time>
+    </div>
+    <aside class="titulosone">
+      <div class="plus">
```
Comment (emolinaglobant) [inline tatiana-gomez-restrepo/tarea-1/empire.html:29]: @tatisgom27 , at this point you should be using semantic tags not divs

**Comment by emolinaglobant** [comment]
@tatisgom27 , the cover does not look right, the layout is off and your not using Flex or Grid to build the layout.

Screen Shot 2022-01-24 at 10 30 25 AM

File: tatiana-gomez-restrepo/tarea-1/CSS/estilos.css (hunk 1 above)
Comment (tatisgom27) [reply tatiana-gomez-restrepo/tarea-1/CSS/estilos.css:10]: super realizare los cambios

**Review by emolinaglobant — APPROVED** [review]

**Pull request merged**